src/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -209,7 +224,9 @@
    :node-purge-time (timer admin-metrics-registry ["node-purge-time"])
    :report-purge-time (timer admin-metrics-registry ["report-purge-time"])
    :package-gc-time (timer admin-metrics-registry "package-gc-time")
-   :other-clean-time (timer admin-metrics-registry ["other-clean-time"])})
+   :other-clean-time (timer admin-metrics-registry ["other-clean-time"])
+   :resource-events-purges (counter admin-metrics-registry "resource-event-purges")
```

Comment:
Should we move this counter up to the block of counters?

Reply:
That seems reasonable, and might even consider putting these new ones before "other" in each section, but no strong opinion.